kubernetes/create-config-maps.sh.j2

```diff
@@ -4,19 +4,19 @@
 kubectl create configmap nginx-config-snippets                          \
     --from-file=error-pages.conf=config-maps/nginx/error-pages.conf
 
-{% if nginx_storage_ssl %}
+{% if not nginx_storage_allow_http %}
 kubectl create secret generic nginx-storage-secrets                              \
```

Comment:
You should remove all conditions from this file. SSL is now always enabled.

Reply:
Done.